lib/Migration/Version001000Date20240712000000.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/** @noinspection PhpUnused */
+
+declare(strict_types=1);
+
+namespace OCA\Tables\Migration;
+
+use Closure;
+use OCP\DB\Exception;
+use OCP\DB\ISchemaWrapper;
+use OCP\DB\Types;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version001000Date20240712000000 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 * @throws Exception
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if ($schema->hasTable('tables_tables')) {
+			$table = $schema->getTable('tables_tables');
```

Comment:
```suggestion
		if ($schema->hasTable('tables_columns')) {
			$table = $schema->getTable('tables_columns');
```

Seems like the wrong table